### PR TITLE
Blogapp

### PR DESCRIPTION
**Pull request by acolombo1**
- Make sure that the N+1 problem is solved when fetching all posts and their comments for a user.
- Use Capybara to write integration tests for each view in your project.
  - User index page:
    - I can see the username of all other users.
    - I can see the profile picture for each user.
    - I can see the number of posts each user has written.
    - When I click on a user, I am redirected to that user's show page.
  - User show page:
    - I can see the user's profile picture.
    - I can see the user's username.
    - I can see the number of posts the user has written.
    - I can see the user's bio.
    - I can see the user's first 3 posts.
    - I can see a button that lets me view all of a user's posts.
    - When I click a user's post, it redirects me to that post's show page.
    - When I click to see all posts, it redirects me to the user's post's index page.
  - User post index page:
    - I can see the user's profile picture.
    - I can see the user's username.
    - I can see the number of posts the user has written.
    - I can see a post's title.
    - I can see some of the post's body.
    - I can see the first comments on a post.
    - I can see how many comments a post has.
    - I can see how many likes a post has.
    - I can see a section for pagination if there are more posts than fit on the view.
    - When I click on a post, it redirects me to that post's show page.
  - Post show page:
    - I can see the post's title.
    - I can see who wrote the post.
    - I can see how many comments it has.
    - I can see how many likes it has.
    - I can see the post body.
    - I can see the username of each commentor.
    - I can see the comment each commentor left.

### CODE REVIEW

File: spec/integration/integration_spec.rb [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  subject!(:post) { Post.where(author_id: 1).first }
+
+  before(:all) do
+    driven_by(:selenium_chrome_headless)
+  end
+
+  it 'Shows posts title' do
+    visit("http://localhost:3000/users/#{tom.id}/posts/#{post.id}")
+    expect(page).to have_content('Hello by Tom')
+  end
```
Comment (rotimiazeez) [inline spec/integration/integration_spec.rb:157]: - Some of your tests are failing, kindly fix them to proceed to the next project 😄
Screenshot from 2022-08-23 22-35-16

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team 👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯
- Good commit messages 📖 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/integration/integration_spec.rb (hunk 1 above)
Comment (acolombo1) [reply spec/integration/integration_spec.rb:157]: It's the first test in each describe. None of these happen in my computer or my partner's. What can we do? Can you show the failure reason?, all those 'unknown' don't tell anything. I can look for a workaround maybe, like having an empty 'it' in each describe.

**Review by Lordkaito — APPROVED** [review]
Hi @acolombo1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**